unittests/contracts/eosio.mechanics/mech_actions.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+#
+# EOS Mechanics actions script.
+#
+
+API_URL="http://127.0.0.1:8888"
+WALLET_URL="http://127.0.0.1:8900/"
+WALLET_NAME="eosmechanics"
+WALLET_PASS="password to eosmechanics wallet with benchmark permission key"
+PATH="$PATH:/usr/local/bin:/usr/local/eosio/bin/"
+
+date
+cleos -u "$API_URL" --wallet-url "$WALLET_URL" wallet unlock -n "$WALLET_NAME" --password "$WALLET_PASS"
+cleos -u "$API_URL" --wallet-url "$WALLET_URL" push action eosmechanics cpu '' -p eosmechanics@benchmark -f
+#cleos -u "$API_URL" --wallet-url "$WALLET_URL" push action eosmechanics ram '' -p eosmechanics@benchmark -f
```

Comment:
I believe disabled code should either include a comment indicating why it's disabled and the criteria for re-enabling or be removed. Please document or remove.

Reply:
@ScottBailey This was ported code from AlohaEOS/eos-mechanics repository and it was commented out in that codebase.
https://github.com/AlohaEOS/eos-mechanics/blob/master/scripts/mech_actions.sh